scripts/dev_tools.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import check_call
+
+def lint():
+    check_call(['pylint', '.'])
+
+def test():
+    check_call(['pytest', '--cov=mev_inspect', 'tests'])
+
+def isort():
+    check_call(['isort', '.'])
+
+def isortcheck():
+    check_call(['isort', '--diff', '.'])
+
+def mypy():
+    check_call(['mypy', '.'])
+
+def black():
+    check_call(['black', '.'])
+
+def blackcheck():
+    check_call(['black', '--diff', '--color', '.'])
+
+def start():
+    check_call(['docker', 'run', 'mev-inspect'])
+
+def build():
+    check_call(['docker', 'build', '-t', 'mev-inspect', '.'])
+
+def start_it():
+    check_call(['docker', 'run', 'mev-inspect'])
```

Comment:
can we change these to the docker compose equivalents? https://github.com/flashbots/mev-inspect-py/pull/29

Reply:
using compose now